docs/overview.md

```diff
@@ -0,0 +1,85 @@
+Ansible-NAS currently assumes you know your way around a server. This page is an
+overview for absolute NAS beginners so they can decide if it is right for them.
+
+## The big picture
+
+To start off _really_ simple: A NAS ([Network Attached
+Storage](https://en.wikipedia.org/wiki/Network-attached_storage)) is a server
+mostly for home or other small networks that offers file storage. It's usually a
+small box that sits in the corner and runs 24/7. These days, a NAS doesn't just
+only handle files, but also offers other services, for instance video streaming
+with [Plex](https://www.plex.tv/) or [Emby](https://emby.media/index.html). You
+can buy consumer NAS boxes from [various
+manifacturers](https://en.wikipedia.org/wiki/List_of_NAS_manufacturers) where
+you just have to add the hard drives, or you can configure your own hardware and
+use open-source software as the operating system. 
+
+One example of the second variant you'll see mentioned here is
+[FreeNAS](https://freenas.org/). It is based on
+[FreeBSD](https://www.freebsd.org/), which like Linux belongs to the family of
+Unix-like operating systems. One strength of FreeBSD/FreeNAS is that it  
+includes the powerful ZFS file system
+([OpenZFS](http://www.open-zfs.org/wiki/Main_Page), to be exact).  However, it
+does not support the [Docker](https://www.docker.com/) containers the way Linux
+does. Also, the Linux ecosystem is larger. On the other hand, very few Linux
+distributions include ZFS out of the box because of licensing issues.
+
+Ansible-NAS in its default form attempts to have the best of both worlds by
+using Docker on Linux with ZFS. This is possible because the
+[Ubuntu](https://www.ubuntu.com/server) Linux distribution supports both
+technologies. As the name says, Ansible-NAS uses
+[Ansible](https://www.ansible.com/) server automation which is usually deployed
+on big multi-machine enterprise systems, not small home servers the size of a
+breadbox. 
+
+## Before you take the plunge
+
+The commercial NAS vendors try to make setting up and running a NAS as simple
+and painless as possible - for a fee, obviously. The open-source NAS software
+providers have lots of resources to help you get started with your own hardware.
+FreeNAS for instance comes with extensive documentation, good introductions to
+ZFS and other topics, and a large community to lean on. 
+
+With Ansible-NAS, at this point at least, you're pretty much on your own. Though
+there is a [Gitter](https://gitter.im/Ansible-NAS/Chat) chat room (see
+[support](support.md)), you're expected to have some familiarity with the
+technologies involved and be able to set up the basic stuff yourself. 
+
+As a to-do list, before you can even install Ansible-NAS, you'll have to:
+
+1. Choose, buy, configure, and test your own **hardware**. Note that ZFS eats
+   RAM like a pig, prefers to have the hard drives all to itself, and should be
```

Comment:
I think it may be worth clarifying ZFS memory requirements as it depends massively on the use case, or perhaps linking something for people to reference. ie, I run my Ansible-NAS on a poxy old AMD with 16gb memory, and it's fine, even with tonnes of Docker containers running.

Same for ECC memory - is this relevant for the home user? It depends on your use case :-) I happen to agree with you, but is someone that's download torrents and saving tv shows to disk going to care?

Reply:
Good point. I'll add some more stuff on that.